### ISSUE TITLE
Upgrade bug

### ISSUE THREAD

**Issue by Miccro** [Anahkiasen/cookie-monster#29]
Not sure if it's just me but everytime I try to to buy an upgrade with cookie monster running it takes the cookies from me but doesn't give me the upgrade. It seemed to be working fine yesterday it's just today. I'll do some more testing see if it might be something else.

**Comment by worldendDominator** [comment]
Does the upgrade stay in the store?
It happened to me a few times that I clicked an upgrade but it didn't purchase, but when I clicked a second (sometimes third) time, it did. I thought the game just "missed" my click (it's pretty laggy), dont know if any cookies were lost.

**Comment by Miccro** [comment]
Yeah, the upgrade stays in the store but I do definitely lose the cookies, not so bad with cheap upgrades but bit annoying when it costs a few hundred trillion.

**Comment by Kenny85** [comment]
Same here, happens once in a while. Now I can't buy Santa's Dominion upgrade, I lose the cookies, but the upgrade remains in the list. Same goes for the Season Savings upgrade.

edit: Fixed it for now by not running the script and then buying the upgrades
